packages/engine/src/workerpool/mod.rs

```diff
@@ -249,7 +249,8 @@ impl WorkerPoolController {
                     })?;
                 }
                 let fut = async move {
-                    let sync = sync; // Capture `sync` in lambda.
+                    let sync = sync;
+                    // Capture `sync` in lambda.
```

Comment:
Can't tell why this happens all the time. 😒 
```suggestion
                    let sync = sync; // Capture `sync` in lambda.
```